draft-ietf-ppm-dap.md

```diff
@@ -1223,7 +1224,10 @@ Clients SHOULD NOT upload the same measurement value in more than one report if
 the Leader responds with HTTP status code 201 Created.
 
 If the Leader does not recognize the task ID, then it MUST abort with error
-`unrecognizedTask`.
+`unrecognizedTask`. If the task has expired, i.e., `task_start + task_duration`
+is greater than the current time, then it SHOULD abort with error
```

Comment:
Aside from my other comment, I don't see why this is a SHOULD whereas aborting with `unrecognizedTask` is a MUST for other cases of unrecognized task ID.